beacon-chain/db/filesystem/metrics.go

```diff
@@ -24,9 +24,9 @@ var (
 		Name: "blob_pruned_blobs_total",
 		Help: "Total number of pruned blobs.",
 	})
-	blobsTotalGauge = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
I'm abandoning the goal of trying to track the total number of blobs on disk. If we track blobs written and blobs pruned, people should be able to deal with that in metrics. But I just noticed, this is a gauge, needs to change to a counter.

Reply:
gauge fixed

Reply:
You wouldn't know the total number of blobs from blobs written / pruned since these values start at zero when you start up the process. There may be blobs from a prior run on disk.

Why are we giving up on tracking total number of blobs on disk?

Reply:
I was thinking about this wrong, you're right that we need a gauge to track the total. We can set the gauge in the cache warmup, the `Save` increments it, `prune` decrements it. But that requires some refactors that I would like to punt to the next PR.